crates/wasm-compose/tests/compositions/arg-not-satisified/error.txt

```diff
@@ -0,0 +1 @@
+import `a` for instantiation `b` cannot be satisfied by dependency `a`
```

Comment:
At least in the abstract if this is the complete error this seems like it could be improved. I presume the error here is that the exported function `a` is missing from the import, and mentioning that in the error message might help make it more clear

Reply:
Indeed, the error message should definitely reflect, at the very least, that it can't be satisfied because it is _incompatible_.

Reply:
I think I'll update the subtype check to be more of a "verify that this is a subtype of" (i.e. `Result<()>`) rather than a simple "is this a subtype of" (i.e. `bool`).

This should help explain why the dependency is incompatible.

Reply:
Ok hopefully the error message is a little more clear here.